operator-wg/whitepaper/052_metacontroller.md

```diff
@@ -0,0 +1,29 @@
+## Metacontroller - Lightweight Kubernetes controllers as a service
```

Comment:
I'm not sure if this is possible, but I think it would be nice to get an example what can be done and how this might look like in metacontroller (in a conceptual way like an image or if its easier by some code examples) ...

Reply:
added example, if you could give your impression, both me and @AmitKumarDas are maintainers of metacontroller, so it can be understandable by us, but maybe not so clear for people not involved, which are target of this whitepaper.